packages/core/src/types/plugins.ts

```diff
@@ -10,6 +10,12 @@ interface InsertParams {
 
 export type PluginTypeI = 'dataCollect' | 'dataAccess' | 'dataProcess' | 'datasetProcess' | 'modelLoad' | 'modelDefine' | 'modelTrain' | 'modelEvaluate';
 
+// TODO(feely): remove 'fs'
```

Comment:
Why do we remove the fs support?

Reply:
OK, the `fs` protocol is for tarball-stream which upload from client, rename to `tarball-stream`?

Reply:
Protocol is the concept that needs to be passed to the user,`fs` is the implementation, should it be changed to a more understandable name, like `tarball-stream` or `local-tarball`?

Reply:
Ok, you should update what you commented before in the TODO.